tests/acceptance/features/collaborativePosix/collaborativePosixFS.feature

```diff
@@ -0,0 +1,13 @@
+Feature: create a resources using collaborative posixfs
```

Comment:
should we add some tags here to make sure these tests only run when `STORAGE_USERS_POSIX_WATCH_FS` is set to `true`?